substrate/network/src/test/sync.rs

```diff
@@ -19,6 +19,7 @@ use sync::SyncState;
 use super::*;
 
 #[test]
+#[ignore]
```

Comment:
all tests in this file are ignored right now

Reply:
they take multiple minutes on my fast laptop to run. that's a pain.

Reply:
usually because `test_runtime/genesis` is outdated so it falls back to wasm executor

Reply:
still, i'd prefer for CI to have these enabled rather that running them every time.